bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/i18n/netatmo.properties

```diff
@@ -398,11 +398,13 @@ request-time-out = Request timed out - will attempt reconnection later
 
 # actions
 
+actionSetThermRoomTempLabel = send temperature setpoint to the thermostat for a room
+actionSetThermRoomTempDesc = Sends a temperature setpoint (and switch to manual mode) to the thermostat for a room with an end time.
 actionInputSetpointLabel = Setpoint
 actionInputSetpointDesc = The temperature setpoint
 actionInputEndtimeLabel = Endtime
 actionInputEndtimeDesc = Time the setpoint should end
+actionSetThermRoomModeLabel = send mode to the thermostat for a room
+actionSetThermRoomModeDesc = Sends a mode to the thermostat for a room with an optional end time.
 actionInputModeLabel = Mode
 actionInputModeDesc = The mode to set: MANUAL, SCHEDULE or FG (Frost-Guard)
```

Comment:
As previously mentioned, this description is not coherent with the code. Waiting for clarification.